app/assets/javascripts/app/clone_account_memberships.coffee

```diff
@@ -0,0 +1,17 @@
+$(document).ready ->
+
+  # Select all for checkboxes
+  selectAllSelector = $("a#selectAllClone")
```

Comment:
I think we have some common logic that we can use for this. I'll have to find it.